### PR TITLE
 HW7 JDI

### PR DESCRIPTION
**Pull request by solomencev**
image

### CODE REVIEW

File: src/main/java/com/epam/tc/hw7/pages/JdiHomePage.java [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+import com.epam.tc.hw7.Config;
+import com.epam.tc.hw7.forms.LoginForm;
+import jdi.entities.User;
+
+public class JdiHomePage extends WebPage {
+
+    public static final User ROMAN_IOVLEV = new User(
+        Config.getUserNameFromProperties(),
+        Config.getUserPasswordFromProperties(),
+        Config.getUserFullNameFromProperties());
```
Comment (annav00) [inline src/main/java/com/epam/tc/hw7/pages/JdiHomePage.java]: There can be many different users, they refer to the implementation of the tests, not the implementation of the page.

File: src/test/java/com/epam/tc/hw7/JdiSiteTest.java [hunk 2]
```diff
@@ -0,0 +1,16 @@
+package com.epam.tc.hw7;
+
+import static com.epam.tc.hw7.pages.JdiHomePage.ROMAN_IOVLEV;
+
+import org.testng.annotations.Test;
+
+public class JdiSiteTest extends AbstractTest {
+
+    @Test(dataProvider = "DataProviderJson", dataProviderClass = DataProviderMetalsAndColors.class)
+    public void jdiHomepageTest(jdi.entities.MetalsAndColors metalsAndColors) {
```
Comment (annav00) [inline src/test/java/com/epam/tc/hw7/JdiSiteTest.java]: Not the best test name. Try to come up with something more explicitly describing the essence.

**Review by annav00 — CHANGES_REQUESTED** [review]

File: src/main/java/com/epam/tc/hw7/pages/JdiHomePage.java (hunk 1 above)
Comment (solomencev) [reply src/main/java/com/epam/tc/hw7/pages/JdiHomePage.java]: Moved creation of user to AbstractTest

File: src/test/java/com/epam/tc/hw7/JdiSiteTest.java (hunk 2 above)
Comment (solomencev) [reply src/test/java/com/epam/tc/hw7/JdiSiteTest.java]: Added description and renamed test for HM7

**Review by annav00 — APPROVED** [review]

**Pull request merged**